src/TestSuite/TestCase.php

```diff
@@ -249,9 +250,56 @@ protected function tearDown(): void
         $this->_configure = [];
         $this->_tableLocator = null;
 
-        if ($this->fixtureManager) {
-            $this->fixtureManager->teardownTest($this);
-            $this->fixtureManager = null;
+        $this->teardownFixtures();
+    }
+
+    /**
+     * Initialized and loads any use fixtures.
+     *
+     * @return void
+     */
+    protected function setupFixtures(): void
+    {
+        $fixtureNames = $this->getFixtures();
+        if (empty($fixtureNames)) {
+            return;
+        }
+
+        if (!isset(static::$schemaStrategies[$this->schemaStrategyClass])) {
+            $schemaStrategy = new $this->schemaStrategyClass();
+            if (!$schemaStrategy instanceof SchemaStrategyInterface) {
+                throw new RuntimeException(
+                    "Schema strategy `{$this->schemaStrategyClass}` does not implement `SchemaStrategyInterface`."
+                );
+            }
+            $schemaStrategy->load();
+            static::$schemaStrategies[$this->schemaStrategyClass] = $schemaStrategy;
+        }
```

Comment:
Can't a similar result be achieved by loading schema in `tests/bootstrap.php`?

Reply:
That's up to you. This lets people just "set and run", but they will require setting up schema data so it isn't anywhere close to a free implementation.

I'm not sold on the schema strategy if it's not needed.